tests/FSharp.Data.GraphQL.Tests/Relay/NodeTests.fs

```diff
@@ -82,9 +133,9 @@ let ``Node with global ID gets correct record`` () =
     let expected2 =    
       NameValueLookup.ofList [
         "model", upcast "Tesla S" ]
-    execAndValidateNode query1 expected1
+    execAndValidateNode query2 expected2 None
```

Comment:
This test was broken, validating two times the first case, instead of the two cases.